dask_ml/linear_model/glm.py

```diff
@@ -230,7 +231,10 @@ def predict(self, X):
         C : array, shape = [n_samples,]
             Predicted class labels for each sample
         """
-        return self.predict_proba(X) > 0.5  # TODO: verify, multi_class broken
+        prob = self.predict_proba(X)
+        if self.n_classes==2:
+            return prob.argmax(axis=1)
+        return prob>0.5 # TODO: verify, multi_class broken
```

Comment:
This comment is no longer needed?

Reply:
Multi-class is still broken, right ? I have solved the issue when there are only two classes and returning the output similar to the sklearn module  .